builtin/providers/scaleway/provider.go

```diff
@@ -10,17 +10,17 @@ import (
 func Provider() terraform.ResourceProvider {
 	return &schema.Provider{
 		Schema: map[string]*schema.Schema{
-			"access_key": &schema.Schema{
```

Comment:
please keep `access_key` here and mark it as `Deprecated: true,`, for backwards compatability. 

e.g. 

```
"access_key": &schema.Schema{
  Deprecated: true,
},
"token": &schema.Schema{
},
…
```